features/step_definitions/shared/xcode.rb

```diff
@@ -22,3 +22,16 @@
 
   xcodebuild(*args)
 end
+
+Then(/the scheme (.+) has a build setting (.+) with value (.+) for the configuration (.+)/) do |scheme, key, value, config|
+  args = [
+    "-scheme", scheme,
+    "-workspace", @workspace_path,
+    "-configuration", config,
+    "-showBuildSettings"
+]
+
+  out, err, status = Open3.capture3("xcodebuild", *args)
+  search_for = "#{key} = #{value}"
```

Comment:
If the underlying `xcodebuild` command fails, the error that we'd get with this logic complicates debugging the issue. I suggest adding the following:

```ruby
flunk(err) unless status.success?
```
That way we fail the test if the `xcodebuild` command fails.